controllers/opentelemetrycollector_controller.go

```diff
@@ -246,10 +202,144 @@ func (r *OpenTelemetryCollectorReconciler) RunTasks(ctx context.Context, params
 			}
 		}
 	}
+	return nil
+}
+
+func (r *OpenTelemetryCollectorReconciler) getParams(instance v1alpha1.OpenTelemetryCollector) manifests.Params {
+	return manifests.Params{
+		Config:   r.config,
+		Client:   r.Client,
+		Instance: instance,
+		Log:      r.log,
+		Scheme:   r.scheme,
+		Recorder: r.recorder,
+	}
+}
+
+// BuildAll returns the generation and collected errors of all manifests for a given instance.
+func (r *OpenTelemetryCollectorReconciler) BuildAll(params manifests.Params) ([]client.Object, error) {
+	builders := []manifests.Builder{
+		collector.Build,
+		targetallocator.Build,
+	}
+	var resources []client.Object
+	for _, builder := range builders {
+		objs, err := builder(params)
+		if err != nil {
+			return nil, err
+		}
+		resources = append(resources, objs...)
+	}
+	return resources, nil
+}
+
+func (r *OpenTelemetryCollectorReconciler) doCRUD(ctx context.Context, params manifests.Params) error {
+	// Collect all objects owned by the operator, to be able to prune objects
+	// which exist in the cluster but are not managed by the operator anymore.
```

Comment:
Can we move this commend on top of the method? (Then it gets displayed in the IDE popup :rabbit: )